File: Aula08-Exercício/Notebook.py
from Computador import Computador

class Notebook(Computador):

    def __init__(self, modelo, ano, preco, tempodebateria):
        super().__init__(modelo, ano, preco)
        self.__tempoDeBateria = tempodebateria
    
    
    def cadastrar(self):
        print("Notebook cadastrado")
        


    def getInformacoes(self):
        return super().getInformacoes() + f" , {self.__tempoDeBateria}"
    

File: Aula09-ExercícioEmJava/Notebook.java
public class Notebook extends Computador {
    
    private int tempoDeBateria;

    public Notebook(){
        super();
    }

    public Notebook(String modelo, String cor, double preco, int tempo){
        super( modelo, cor, preco );
        this.tempoDeBateria = tempo;
    }

    @Override
    public void cadastrar(){
        System.out.println("Notebook" + this.modelo + " cadastrado");
    }

    @Override
    public String getInformacoes() {
        return super.getInformacoes() + "\nTempo de Bateria: " + this.tempoDeBateria;
    }

    @Override
    public String toString() {
        return getInformacoes();
    }

}
